README.md

```diff
@@ -1 +1,33 @@
-# functional-harness
+#### functional-harness
+
+Allows to run functional tests in local browsers or on SauceLabs.
+
+##### What it's doing
+
+1) Starts server for test pages.
+2) Opens the specified browsers (locally or on SauceLabs).
+3) Run mocha tests.
+4) Closes the browsers and stop the server after the tests.
```

Comment:
1) Starts the server with the test pages.
2) Opens the specified browsers (locally or on SauceLabs).
3) Runs mocha tests.
4) Closes the browsers and stops the server after all tests are finished.